UACloudLibraryServer/Startup.cs

```diff
@@ -142,12 +165,26 @@ public void ConfigureServices(IServiceCollection services)
                             }
                         };
                     });
+            }
+
+#if !NO_AZURE_AD
+            if (Configuration.GetSection("AzureAd")?["ClientId"] != null)
+            {
+                services.AddAuthentication()
+                    .AddMicrosoftIdentityWebApp(Configuration, configSectionName: "AzureAd", openIdConnectScheme: "AzureAd", subscribeToOpenIdConnectMiddlewareDiagnosticsEvents: true, displayName: "CESMII Test")
```

Comment:
Remove "Test"?

Reply:
Yes, this is all still a draft/work in progress. Just created the PR to make sure the GitHub actions still work.